Sources/LogConsole/LogInterceptor.cs

```diff
@@ -10,6 +10,8 @@
 using System;
 using UnityEngine;
 
+using Logger = KSPDev.LogUtils.Logger;
```

Comment:
`KSPDev.LogUtils.Logger` is a pre-1.1 utility class. It should be deprecated in favor of `Debug.LogXXXFormat()` everywhere.

Anyways, thanks to take care of it!